src/util/FileSystemService.java

```diff
@@ -0,0 +1,75 @@
+package util;
+
+import data.Record;
+
+import java.io.*;
+import java.util.LinkedList;
+import java.util.List;
+
+public class FileSystemService {
+
+    private static final String NAME_OF_FILE = "C:\\Java47\\PasswordReminder\\passRem.txt";
+
+    private void createFile() {
+        File f = new File(NAME_OF_FILE);
+        if (!f.exists()) {
+            try {
+                f.createNewFile();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    private List<Record> readFromFile() {
+        List<Record> lr = new LinkedList<>();
+        try (ObjectInputStream ois = new ObjectInputStream(new FileInputStream(NAME_OF_FILE))) {
```

Comment:
Not use NAME_OF_FILE, use output of createFile() method for this here and in other places